### PR TITLE
Norwegian translations added again

### PR DESCRIPTION
**Pull request by Vikingn**
Now I think I have understand how to do this !

Does it look nice now ?

### CODE REVIEW

**Comment by MichaelDvP** [comment]
Yes, now it's the right base. I think you have also made changes to the web index.ts, please add this file. And in line 119 for "secondes" you have edited the column of pl-language.

**Comment by Vikingn** [comment]
Fixed line 119  Used Google translate for the polish seconds I have just edited this file

**Comment by Vikingn** [comment]
Hi !

Can you look at it now ?

It should just be the locale_translations.h file.

I have changed line 119 so the polish should be right too.

Sendt fra E-post<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

Fra: ***@***.***>
Sendt: onsdag 22. mars 2023 kl. 16.10
Til: ***@***.***>
Kopi: ***@***.***>; ***@***.***>
Emne: Re: [emsesp/EMS-ESP32] Norwegian translations added again (PR #1138)


Yes, now it's the right base. I think you have also made changes to the web index.ts, please add this file. And in line 119 for "secondes" you have edited the column of pl-language.

**Review by MichaelDvP — CHANGES_REQUESTED** [review]
I've commented in the changes. Rest looks good.

**Comment by Vikingn** [comment]
Hi !

Sekund was it before I edited it now.

Google translate say Sekundy

I thaught something was wrong from line 271, and asked proddy.. took some days and I missed to add those back.

Should I fix those lines or do you do it ?



Sendt fra E-post<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

Fra: ***@***.***>
Sendt: onsdag 22. mars 2023 kl. 17.04
Til: ***@***.***>
Kopi: ***@***.***>; ***@***.***>
Emne: Re: [emsesp/EMS-ESP32] Norwegian translations added again (PR #1138)


@MichaelDvP requested changes on this pull request.

I've commented in the changes. Rest looks good.


In src/locale_translations.h<https://github.com/emsesp/EMS-ESP32/pull/1138#discussion_r1145051325>:

> @@ -116,19 +116,19 @@ MAKE_WORD_TRANSLATION(OFF, "OFF", "AUS", "UIT", "AV", "wył.", "AV", "OFF", "KAP

 MAKE_WORD_TRANSLATION(minutes, "minutes", "Minuten", "Minuten", "Minuter", "minut", "Minutter", "minutes", "dakika")

 MAKE_WORD_TRANSLATION(hours, "hours", "Stunden", "Uren", "Timmar", "godzin", "Timer", "heures", "saat")

 MAKE_WORD_TRANSLATION(days, "days", "Tage", "Dagen", "Dagar", "dni", "Dager", "jours", "gün")

-MAKE_WORD_TRANSLATION(seconds, "seconds", "Sekunden", "Seconden", "Sekunder", "sekund", "Sekunder", "secondes", "saniye")

+MAKE_WORD_TRANSLATION(seconds, "seconds", "Sekunden", "Seconden", "Sekunder", "sekundy", "Sekunder", "secondes", "saniye")

i think the old pl-translation was correct: "sekund"


In src/locale_translations.h<https://github.com/emsesp/EMS-ESP32/pull/1138#discussion_r1145054826>:

> @@ -268,487 +268,487 @@ MAKE_WORD_TRANSLATION(cyl2, "cyl 2", "Zyl_2", "Cil 2", "Cyl 2", "cyl 2", "cyl 2"



 // Entity translations

 // Boiler

-MAKE_TRANSLATION(wwtapactivated, "wwtapactivated", "turn on/off", "Durchlauferhitzer aktiv", "zet aan/uit", "på/av", "system przygotowywania", "slå på/av", "ecs activée")

-MAKE_TRANSLATION(reset, "reset", "Reset", "Reset", "Reset", "Nollställ", "kasowanie komunikatu", "nullstill", "reset")

-MAKE_TRANSLATION(oilPreHeat, "oilpreheat", "oil preheating", "Ölvorwärmung", "Olie voorverwarming", "Förvärmning olja", "podgrzewanie oleju", "oljeforvarming", "préchauffage de l'huile")

-MAKE_TRANSLATION(heatingActive, "heatingactive", "heating active", "Heizen aktiv", "Verwarming actief", "Uppvärmning aktiv", "c.o. aktywne", "oppvarming aktiv", "chauffage actif")

-MAKE_TRANSLATION(tapwaterActive, "tapwateractive", "tapwater active", "Warmwasser aktiv", "Warm water actief", "Varmvatten aktiv", "c.w.u. aktywne", "varmtvann aktiv", "eau chaude active")

for these lines 271-275 you have removed the en-fullname, not good.

File: src/locale_translations.h [hunk 1]
```diff
@@ -116,19 +116,19 @@ MAKE_WORD_TRANSLATION(OFF, "OFF", "AUS", "UIT", "AV", "wył.", "AV", "OFF", "KAP
 MAKE_WORD_TRANSLATION(minutes, "minutes", "Minuten", "Minuten", "Minuter", "minut", "Minutter", "minutes", "dakika")
 MAKE_WORD_TRANSLATION(hours, "hours", "Stunden", "Uren", "Timmar", "godzin", "Timer", "heures", "saat")
 MAKE_WORD_TRANSLATION(days, "days", "Tage", "Dagen", "Dagar", "dni", "Dager", "jours", "gün")
-MAKE_WORD_TRANSLATION(seconds, "seconds", "Sekunden", "Seconden", "Sekunder", "sekund", "Sekunder", "secondes", "saniye")
+MAKE_WORD_TRANSLATION(seconds, "seconds", "Sekunden", "Seconden", "Sekunder", "sekundy", "Sekunder", "secondes", "saniye")
```
[the remark this answers is not in the document]
Comment (Vikingn) [reply src/locale_translations.h:119]: sekund is norwegian for 1 second. sekundy is seconds in polish say google translate

**Comment by MichaelDvP** [comment]
> Should I fix those lines or do you do it ?

After all, you are only 6 lines away  from a successful PR, i think now bring it to the good end.

**Comment by Vikingn** [comment]
I do the lines now



Sendt fra E-post<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

Fra: ***@***.***>
Sendt: onsdag 22. mars 2023 kl. 17.17
Til: ***@***.***>
Kopi: ***@***.***>; ***@***.***>
Emne: Re: [emsesp/EMS-ESP32] Norwegian translations added again (PR #1138)


Should I fix those lines or do you do it ?

After all, you are only 6 lines away from a successful PR, i think now bring it to the good end.

**Comment by Vikingn** [comment]
Edited the file again fixed line 271-275

File: src/locale_translations.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (MichaelDvP) [reply src/locale_translations.h:119]: Afair pl translation is made by @pswid  I think as native speaker he knows better than google translate. Or, if this is wrong, he will correct it.

**Comment by Vikingn** [comment]
Hi !
It was strange, because you said polish was wrong, and now you want it back to what it was ?

I think maybe I could have changed it by a mistake before.

I think an easy Word as Seconds should be right in Google Translate.

But do as you want ! Im shure that the polish person know polish much better than me.

Sendt fra E-post<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

Fra: ***@***.***>
Sendt: onsdag 22. mars 2023 kl. 17.39
Til: ***@***.***>
Kopi: ***@***.***>; ***@***.***>
Emne: Re: [emsesp/EMS-ESP32] Norwegian translations added again (PR #1138)


@MichaelDvP commented on this pull request.


In src/locale_translations.h<https://github.com/emsesp/EMS-ESP32/pull/1138#discussion_r1145122304>:

> @@ -116,19 +116,19 @@ MAKE_WORD_TRANSLATION(OFF, "OFF", "AUS", "UIT", "AV", "wył.", "AV", "OFF", "KAP

 MAKE_WORD_TRANSLATION(minutes, "minutes", "Minuten", "Minuten", "Minuter", "minut", "Minutter", "minutes", "dakika")

 MAKE_WORD_TRANSLATION(hours, "hours", "Stunden", "Uren", "Timmar", "godzin", "Timer", "heures", "saat")

 MAKE_WORD_TRANSLATION(days, "days", "Tage", "Dagen", "Dagar", "dni", "Dager", "jours", "gün")

-MAKE_WORD_TRANSLATION(seconds, "seconds", "Sekunden", "Seconden", "Sekunder", "sekund", "Sekunder", "secondes", "saniye")

+MAKE_WORD_TRANSLATION(seconds, "seconds", "Sekunden", "Seconden", "Sekunder", "sekundy", "Sekunder", "secondes", "saniye")

Afair pl translation is made by @pswid<https://github.com/pswid> I think as native speaker he knows better than google translate. Or, if this is wrong, he will correct it.

**Comment by proddy** [comment]
this is still missing the index.ts file @Vikingn

**Comment by Vikingn** [comment]
Why do I need to send that file ? I dont think I have changed that file
It look like this:
export * from './ap';
export * from './features';
export * from './me';
export * from './mqtt';
export * from './ntp';
export * from './security';
export * from './signin';
export * from './system';
export * from './network';

**Comment by proddy** [comment]
I meant the web translations from `\interface\src\i18n\no\index.ts`  like the fix for Days

**Comment by Vikingn** [comment]
Tranlated what was missing and created a new PR 1140

Sendt fra E-post<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

Fra: ***@***.***>
Sendt: torsdag 23. mars 2023 kl. 21.48
Til: ***@***.***>
Kopi: ***@***.***>; ***@***.***>
Emne: Re: [emsesp/EMS-ESP32] Norwegian translations added again (PR #1138)


I meant the web translations from \interface\src\i18n\no\index.ts like the fix for Days

**Comment by Vikingn** [comment]
Good, I can send the inder file later today

Sendt fra min iPhone

24. mar. 2023 kl. 12:14 skrev Proddy ***@***.***>:

﻿

Merged #1138<https://github.com/emsesp/EMS-ESP32/pull/1138> into dev.

**Pull request merged**